platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java

```diff
@@ -217,6 +218,28 @@ public static String hex(final byte[] bytes, final int length) {
         return new String(out);
     }
 
+    /**
+     * Converts Bytes to a lowercase hexadecimal string.
+     *
+     * @param bytes  the bytes to hexadecimal
+     * @param length the length of the array to convert to hex
+     * @return a {@link String} containing the lowercase hexadecimal representation of the byte array
+     */
+    public static String hex(final Bytes bytes, final int length) {
```

Comment:
BTW, couldn't we use Bytes.toHex() instead of this utility method?

Reply:
Bytes.toHex() does not allow us to specify the length. I could convert the whole thing to hex and take a substring if that's what you prefer

Reply:
Alternatively, you could check if anyone uses toHex() already, and if not, perhaps you could move the length implementation into that method so that it finally becomes useful.

But I'm okay either way.